dlc-node/src/main/scala/org/bitcoins/dlc/node/config/DLCNodeAppConfig.scala

```diff
@@ -55,6 +55,16 @@ case class DLCNodeAppConfig(
     new InetSocketAddress(uri.getHost, uri.getPort)
   }
 
+  lazy val externalIP: Option[InetSocketAddress] = {
+    val path = s"bitcoin-s.$moduleName.external-ip"
+    if (config.hasPath(path)) {
+      val str = config.getString(path)
+      Some(NetworkUtil.parseInetSocketAddress(str, listenAddress.getPort))
```

Comment:
the port should probably be configurable in case they are doing some reverse proxy or something

Reply:
It is configurable `listenAddress.getPort` provides a default value.